sklearn/ensemble/_hist_gradient_boosting/tests/test_compare_lightgbm.py

```diff
@@ -12,6 +12,10 @@
 from sklearn.model_selection import train_test_split
 
 
+# TODO(1.8) remove the filterwarnings decorator
+@pytest.mark.filterwarnings(
+    "ignore:'force_all_finite' was renamed to 'ensure_all_finite':FutureWarning"
+)
```

Comment:
Is this coming from `lightgbm`?

Reply:
Yes, technically from scikit-learn as a dependency of lightgbm.